notebooks/delta_table_upload_demo.ipynb

```diff
@@ -0,0 +1,492 @@
+      "source": [
+        "#Make sure the Delta table saved properly\n",
+        "df = spark.read.load(table_path)\n",
+        "df.show()"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "_iXgAkqNkGRP",
+        "outputId": "2d9354dd-fa09-439a-9e8e-0ce406d82a24"
+      },
+      "execution_count": 8,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "+------------------+--------------------+\n",
+            "|        global_key|            row_data|\n",
+            "+------------------+--------------------+\n",
+            "|sample-image-1.jpg|https://lb-test-d...|\n",
+            "+------------------+--------------------+\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#When uploading data from a Delta table saved locally, only the dataset_id and table_path argument are required. The project_id argument\n",
+        "#is only necessary if you want the uploaded data rows to be batched to a project\n",
+        "results = client.create_data_rows_from_delta_table(dataset_id=dataset.uid, project_id=project.uid, verbose=True, table_path=table_path)"
+      ],
+      "metadata": {
+        "id": "K0gpeUq2kf9z"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "We can either create a Spark Session before running `create_data_rows_from_delta_table()`, or we can pass in the `spark_config` argument that the Labelspark Client will use to programatically create the Spark Session."
+      ],
+      "metadata": {
+        "id": "nZQfLzZ8XINp"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# AWS:\n",
+        "\n",
+        "Build Spark Session before running `create_data_rows_from_delta_table()`:"
+      ],
+      "metadata": {
+        "id": "61q1CNPH1vxm"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "builder = SparkSession.\\\n",
+        "        builder.\\\n",
+        "        appName(\"pyspark-test\").\\\n",
+        "        config(\"spark.driver.extraClassPath\", f\"{SPARK_HOME}/jars/hadoop-aws-2.7.4.jar:{SPARK_HOME}/jars/aws-java-sdk-1.7.4.jar\").\\\n",
+        "        config(\"spark.executor.extraClassPath\", f\"{SPARK_HOME}/jars/hadoop-aws-2.7.4.jar:{SPARK_HOME}/jars/aws-java-sdk-1.7.4.jar\")\n",
+        "\n",
+        "spark = configure_spark_with_delta_pip(builder).getOrCreate()\n",
+        "spark._jsc.hadoopConfiguration().set(\"fs.s3a.access.key\", AWS_ACCESS_KEY)\n",
+        "spark._jsc.hadoopConfiguration().set(\"fs.s3a.secret.key\", AWS_SECRET_KEY)\n",
+        "\n",
+        "results = client.create_data_rows_from_delta_table(dataset_id=dataset.uid, project_id=project.uid, verbose=True, table_path=table_path, spark=spark)"
```

Comment:
Do we want results here to be overwritten by `results = client.create_data_rows_from_delta_table(dataset_id=dataset.uid, project_id=project.uid, verbose=True, table_path=table_path, spark_config=aws_spark_config) `in the next cell ?

Reply:
Yeah, the intention here is that only one of the cells with client.create_data_rows_from_delta_table() will be run as this is showing that you can either run the function with a spark argument or spark_config argument